Riot/Modules/Room/RoomViewController.m

```diff
@@ -5525,8 +5525,9 @@ - (void)refreshActivitiesViewDisplay
         }
         else if ([AppDelegate theDelegate].isOffline)
         {
-            self.activitiesViewExpanded = YES;
-            [roomActivitiesView displayNetworkErrorNotification:[VectorL10n roomOfflineNotification]];
+            // Doing nothing here as the affline notification is now handled by the AppCoordinator
+//            self.activitiesViewExpanded = YES;
```

Comment:
Can the commented out code be deleted?

Reply:
I'd like to keep these lines as we may revert to this bottom view in a near future.

Reply:
Hm I have not found commented out code to be useful in practice, though it depends on how long "near future" is. If this is work-in-progress to be changed in the next 2-3 weeks, it can work but for anything longer I find that too much context is lost and it isn't clear anymore why something is commented out and when it should be uncommented. We also have git history for this. But not a blocking issue from my point of view.